lib/opensusebasetest.pm

```diff
@@ -727,6 +727,21 @@ sub wait_grub_to_boot_on_local_disk {
     my @tags = qw(grub2 tianocore-mainmenu);
     push @tags, 'encrypted-disk-password-prompt' if (get_var('ENCRYPT'));
 
+    # Enable boot menu for x86_64 uefi workaround, see bsc#1180080 for details
+    if (is_sle && get_required_var('FLAVOR') =~ /Migration/ && check_var('ARCH', 'x86_64') && get_var('UEFI')) {
+        if (!check_screen(\@tags, 15)) {
+            record_soft_failure 'bsc#1180080';
+            send_key "e";
+            send_key "f2";
+            type_string "exit";
+            send_key "ret";
+            tianocore_select_bootloader;
```

Comment:
Another thinking, would you use a public function to reduce the code redundancy?

Reply:
Two workaround has small different, so i think can keep current state